packages/components/src/utils/math.js

```diff
@@ -19,24 +19,23 @@ export function getNumber( value ) {
 /**
  * Safely adds 2 values.
  *
- * @param {number|string} args Values to add together.
+ * @param {(number|string)[]} args Values to add together.
  *
  * @return {number} The sum of values.
  */
 export function add( ...args ) {
-	return args.reduce( ( sum, arg ) => sum + getNumber( arg ), 0 );
+	return args.map( getNumber ).reduce( ( sum, arg ) => sum + arg, 0 );
```

Comment:
This and the change below should be equivalent in runtime but appeases the typechecking (I couldn't figure out how to communicate to TS through JSDoc that `reduce` was going to be called as `reduce<string|number, number>`. It insisted that the return type of `reduce` be `string|number`. Mapping the array to `number` from `string|number` first avoids this. From a performance perspective I think this is negligible (you'd only start to notice if we were adding several thousand numbers, right?)